arangosh/Import/ImportFeature.cpp

```diff
@@ -201,6 +201,11 @@ void ImportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
                          actionsJoined,
                      new DiscreteValuesParameter<StringParameter>(&_onDuplicateAction, actions));
 
+  options->addOption("--merge-attributes",
+                     "action to merge attributes into one document attribute " +
+                         actionsJoined,
```

Comment:
`actionsJoined` is a string containing `error, update, replace, ignore`, and I don't see any relation between it and the new option.
```suggestion
                     "merge attributes into a new attribute (e.g. mergedAttribute=[someAttribute]-[otherAttribute])",
```